docs/video-miners/core-concepts/winning-ticket-concepts.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: Tickets
+---
+
+The following provides a detailed high level overview about tickets and how their attributes impact orchestrators including but not limited to:
+
+- What are tickets and how do they work
+- How adjusting orchestrator attributes affecting tickets impact orchestator payouts
+- How orchestrator behavior changes with incoming tickets
+
+***Related documentation*** [Earnings](https://docs.livepeer.org/video-miners/core-concepts/earnings) and [Payments](https://docs.livepeer.org/video-miners/core-concepts/payments).
+
+## What is a ticket?
+
+Broadcasters using the Livepeer network attach a “ticket” to each segment of a video they want transcoded. A ticket can be thought of as a form of work completed for a given video stream a broadcaster is emitting to the Livepeer network.  The final number of tickets associated with a given stream is a product of the fees proposed by the orchestrator for doing the work and the total length of the video processed.
+
+A ticket is comprised of the following attributes:
+
+- **recipient**: ETH address of recipient (orchestrator completing transcoding work)
+- **faceValue**: The ETH value of the ticket payable to the recipient (orchestrator) if the ticket wins
+- **winProb**: The probability of the ticket winning (number between 0 and 2^256
+ - 1)
+- **recipientRandHash**: Hashed commitment to a random number recipientRand generated by the recipient
+- **senderNonce**: Random nonce generated by the broadcaster indicating the ticket is unique.
+- **auxData**: Additional data associated with the ticket
+
+Each ticket upon transmission from broadcaster to orchestrator is treated as a “micropayment” worth the expected value of the ticket (calculated as: face value multiplied by the win probability of the ticket).
+
+## How Winning Tickets Are Picked
+
+Picking tickets isn’t as simple as it might seem, we use an implementation based on [probabilistic micropayments](https://medium.com/livepeer-blog/streamflow-probabilistic-micropayments-f3a647672462).  At a high level, each ticket could be thought of as a “lottery” ticket of sorts.  But the raw implementation is a bit more complex than it might seem.
+
+Orchestrators can set a configuration value called ticket expected value or “EV” which defines the value of work an orchestrator is willing to do before requiring a ticket.  By default this value is set to 1000 gwei, which translates to an orchestrator willing to complete 1000 gwei worth of transcoding (based on their pixel pricing configuration) before requiring a ticket.
+
+Winning tickets are selected based on a ticket’s win probability (winProb) and a random value produced by hashing the broadcaster’s random number with a random number from the orchestrator linked to the commitment included in the ticket (recipientHash).  This process is completed at the end of each “round” on the Livepeer network.
```

Comment:
Could you elaborate on this "hashing" function? What is it checking?

Reply:
I abbreviated to not get too far into the weeds, however it could be a good idea to introduce more complexity in a later revision